### PR TITLE
Allow civit ai API Key on Imports

### PR DESCRIPTION
**Pull request by brandonrising**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [ ] Feature
- [x] Bug Fix
- [ ] Optimization
- [ ] Documentation Update
- [ ] Community Node Submission

## Have you discussed this change with the InvokeAI team?
- [x] Yes
- [ ] No, because:

      
## Have you updated all relevant documentation?
- [ ] Yes
- [ ] No

## Description
Small PR to allow users to pass in a civit api key via config options

## Related Tickets & Documents

- Related Issue #
- Closes #

## Added/updated tests?

- [ ] Yes
- [ ] No : _please replace this line with details on why tests
      have not been included_

### CODE REVIEW

**Comment by brandonrising** [comment]
This was just a minimally invasive way to allow users to pass civit api keys for model imports. Unsure of the value prop of adding managing this key to the UI

**Review by lstein — APPROVED** [review]
Looks good to me. Thank you!

File: invokeai/app/services/config/config_default.py [hunk 1]
```diff
@@ -280,6 +280,9 @@ class InvokeAIAppConfig(InvokeAISettings):
     deny_nodes          : Optional[List[str]] = Field(default=None, description="List of nodes to deny. Omit to deny none.", json_schema_extra=Categories.Nodes)
     node_cache_size     : int = Field(default=512, description="How many cached nodes to keep in memory", json_schema_extra=Categories.Nodes)
 
+    # MODEL IMPORT
+    civitai_api_key       : Optional[str] = Field(default=os.environ.get("CIVITAI_API_KEY"), description="API key for CivitAI", json_schema_extra=Categories.Other)
```
Comment (psychedelicious) [inline invokeai/app/services/config/config_default.py:284]: Does everybody set this as an environment variable?

**Review by psychedelicious — APPROVED** [review]

File: invokeai/app/services/config/config_default.py (hunk 1 above)
Comment (brandonrising) [reply invokeai/app/services/config/config_default.py:288]: The environment variable, set it in the config file, or pass it via command line flags.

Comment (brandonrising) [reply invokeai/app/services/config/config_default.py:288]: If it's unset, the config property is just None

**Pull request merged**